src/main/java/WebFilterr.java

```diff
@@ -0,0 +1,46 @@
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import javax.servlet.Filter;
+import java.io.PrintWriter;
+
+@javax.servlet.annotation.WebFilter(filterName = "demofilter",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4535407)